include/openssl/ssl.h

```diff
@@ -5159,6 +5159,9 @@ OPENSSL_EXPORT void SSL_set_tmp_dh_callback(SSL *ssl,
                                             DH *(*cb)(SSL *ssl, int is_export,
                                                       int keylength));
 
+// SSL_CTX_set_dh_auto does nothing.
```

Comment:
Pending my other comment the pattern here seems to be nothing is for void, this technically does one thing: return 1.
```suggestion
// SSL_CTX_set_dh_auto returns 1.
```